Podcast/DiscoverComponentViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  DiscoverComponentViewController.swift
+//  Podcast
+//
+//  Created by Mindy Lou on 2/7/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class DiscoverComponentViewController: ViewController {
```

Comment:
Why do we have `DiscoverComponentViewController` subclassing `ViewController` then `DiscoverViewController` subclassing `DiscoverComponentViewController` when `DiscoverComponentViewController` is never subclassed by anything else .. seems like an unneccary seperation. Did you have another use planned for `DiscoverComponentViewController`?

Reply:
I haven't made the `DiscoverTopicViewController` yet, which basically is almost the same as `DiscoverViewController`, so I wanted to refactor all of the components out so I wouldn't have to create them twice

Reply:
Sounds good! 👍